lib/site-crawler.js

```diff
@@ -36,7 +36,7 @@ export default class SiteCrawler extends EventTarget {
 
       // wait until server has started
       this.serverProcess.stdout.on(`data`, output => {
-        if (output.includes(`Listening on: ${BASE_URL}`)) {
+        if (output.includes(`Listening on:`)) {
```

Comment:
@luc122c Just dropping the check for `${BASE_URL}` does work.
@FloEdelmann Would this be acceptable?

Reply:
I guess it's okay.